### ISSUE TITLE
Travis is currently failing

### ISSUE THREAD

**Issue by zuphilip** [UB-Mannheim/malibu#76]
I don't understand what the problem is...

```
...
Ign:30 http://archive.ubuntu.com/ubuntu wily/universe Translation-en

Err:22 http://archive.ubuntu.com/ubuntu wily/universe amd64 Packages

  404  Not Found [IP: 91.189.88.152 80]

Err:23 http://archive.ubuntu.com/ubuntu wily/universe i386 Packages

  404  Not Found [IP: 91.189.88.152 80]

Ign:27 http://archive.ubuntu.com/ubuntu wily/universe all Packages

Ign:28 http://archive.ubuntu.com/ubuntu wily/universe Translation-en_US

Ign:30 http://archive.ubuntu.com/ubuntu wily/universe Translation-en

Fetched 3,168 B in 3s (1,041 B/s)

Reading package lists...

W: The repository 'http://archive.ubuntu.com/ubuntu wily Release' does not have a Release file.

W: http://dl.hhvm.com/ubuntu/dists/trusty/InRelease: Signature by key 36AEF64D0207E7EEE352D4875A16E7281BE7A449 uses weak digest algorithm (SHA1)

W: http://ppa.launchpad.net/couchdb/stable/ubuntu/dists/trusty/Release.gpg: Signature by key 15866BAFD9BCC4F3C1E0DFC7D69548E1C17EAB57 uses weak digest algorithm (SHA1)

E: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/wily/universe/binary-amd64/Packages  404  Not Found [IP: 91.189.88.152 80]

E: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/wily/universe/binary-i386/Packages  404  Not Found [IP: 91.189.88.152 80]

E: Some index files failed to download. They have been ignored, or old ones used instead.

The command "sudo apt-get update -q" failed and exited with 100 during .

Your build has been stopped.
```

https://travis-ci.org/UB-Mannheim/malibu/builds/278675951

**Comment by stweil** [comment]
There is no Ubuntu wily at the given URL. I'll have a look...

**Comment by zuphilip** [comment]
Fixed  in #77.
